packages/web-app/src/views/components/unorderedList.njk

```diff
@@ -1,9 +1,9 @@
-{% macro unorderedList(items) %}
+{% macro unorderedList(items, rootUrl = '') %}
   <ul class="govuk-list">
 
     {% for item in items %}
-      <li><a class="govuk-link" href="#">{{ item.name }}</a></li>
+      <li><a class="govuk-link" href="{{ rootUrl }}/{{ item.id }}">{{ item.name }}</a></li>
```

Comment:
I can see why you've added  `item.id` to the end of the `href` but I have a slight concern that this could cause us an issue in the future if we want to use this component but don't want the item id added to the end of the URL.

The only other way I can think of doing it is to manipulate the data in the `review-appeal-submission` template before its passed into the component.

We can leave it for now, just something to be aware of.

Reply:
document-service-proxy.spec.js added